source/common/config/well_known_names.cc

```diff
@@ -144,6 +146,10 @@ TagNameValues::TagNameValues() {
   // mongo.(<stat_prefix>.)*
   addTokenized(MONGO_PREFIX, "mongo.$.**");
 
+  if (Runtime::runtimeFeatureEnabled("envoy.restart_features.thrift_stat_prefix_tag_extraction")) {
```

Comment:
Last time I tried this, (IIRC) runtime flags don't work here because of startup-ordering problems. See #17686. cc @jmarantz

Reply:
fun stuff :) let me verify it, because I verified this with the tests

Reply:
It's possible that something has changed since then, and this is no longer a problem. That would be great news.

Reply:
Oh, I see; in my PR, I was using an integration test. I just updated that PR branch onto main and it still fails. I *think* it would also fail on a real (non-test) build due to init-order problems.